documentation/agent/usage.md

```diff
@@ -35,27 +58,47 @@ by sending HTTP requests to the agent.
 
 Example:
 
-    $> fusioninventory-agent --server http://my.server/plugins/fusioninventory/ --daemon
```

Comment:
You're telling later about an existing redirection, but actually this a http server configuration concern and generally speaking I think you shouldn't expect people configure their server with such a redirection. So you should better keep the full URL in example to avoid any confusion.